lib/config/rules/possible-errors.js

```diff
@@ -6,26 +6,15 @@ module.exports = {
   // Enforce return statements in getters
   'getter-return': ['error', {allowImplicit: true}],
   // disallow using an async function as a Promise executor
-  'no-async-promise-executor': 'off',
+  'no-async-promise-executor': 'error',
   // Disallow await inside of loops
   'no-await-in-loop': 'off',
   // Disallow comparing against -0
   'no-compare-neg-zero': 'error',
-  // Disallow or enforce trailing commas
-  'comma-dangle': [
-    'error',
-    {
-      arrays: 'always-multiline',
-      objects: 'always-multiline',
-      imports: 'always-multiline',
-      exports: 'always-multiline',
-      functions: 'always-multiline',
-    },
-  ],
   // Disallow assignment in conditional expressions
   'no-cond-assign': 'error',
   // Disallow use of console
-  'no-console': 'error',
+  'no-console': 'off',
```

Comment:
I have a, "some men just want to watch the world burn" fascination for seeing how this plays out... _but_, if we're following the spirit of the migration guide:
- This should still be enabled by default
- It should be disabled for [the node config](https://github.com/Shopify/eslint-plugin-shopify/blob/master/lib/config/node.js)

Reply:
😬 https://github.com/Shopify/eslint-plugin-shopify/pull/332